packages/gatsby/src/bootstrap/load-plugins/load.ts

```diff
@@ -118,9 +119,17 @@ export function resolvePlugin(
       version: packageJSON.version,
     }
   } catch (err) {
-    throw new Error(
-      `Unable to find plugin "${pluginName}". Perhaps you need to install its package?`
-    )
+    if (process.argv.includes(`--verbose`)) {
```

Comment:
I think you can also check against `gatsby_log_level`: https://github.com/gatsbyjs/gatsby/blob/1baed444e5573a679fd6327741f59299e6b714bc/packages/gatsby-cli/src/create-cli.ts#L112